styles/landing.css

```diff
@@ -0,0 +1,109 @@
+@import url("./main.css");
+
+.products-nav {
+  display: flex;
+  justify-content: space-around;
+}
+
+.product {
+  font-weight: bold;
+  padding: 10px;
+}
+
+.product:hover {
+  background-color: var(--primary-color);
+  color: var(--secondary-color);
+}
+
+.section-b {
+  background-image: url("https://c4.wallpaperflare.com/wallpaper/984/693/125/wheel-tires-wheels-company-wallpaper-preview.jpg");
+  height: 70vh;
+  background-size: cover;
+  background-position: center;
+  margin-top: 1rem;
+}
+
+.intro {
+  color: var(--secondary-color);
+  padding-top: 2rem;
+  padding-left: 6rem;
+}
+
+.main-title {
+  margin-bottom: 2rem;
+  font-size: 1.8rem;
+  text-transform: uppercase;
```

Comment:
This line is also used multiple time so you can create a utility class as txt-uppercase and use it in the HTML